php-practice/section6.php

```diff
@@ -164,9 +191,18 @@ function getTriangleNumber($x) {
  * @return int
  */
 function sum1To100Evens() {
-  // YOUR CODE HERE
+  $value = 0;
+
+    for($i = 0; $i < 100; $i+=2) {
```

Comment:
Missed one thing -- it should be `$i <= 100`. Just doing `<` will loop for `94`, `96`, and end on `98` because `100` is not less than `100`. Make sense?